src/views/dependencyDataProvider.ts

```diff
@@ -124,6 +124,14 @@ export class DependencyDataProvider implements TreeDataProvider<ExplorerNode> {
         const children = (!this._rootItems || !element) ?
             await this.getRootNodes() : await element.getChildren();
 
+            if (children && element instanceof ContainerNode) {
+                if (element.isMavenType()) {
+                    children.sort((a, b) => {
+                        return a.getDisplayName().localeCompare(b.getDisplayName());
```

Comment:
@jdneo, we will still need displayName or label method to be available here, as we need to sort tha array.

Reply:
Makes sense. I now think that sorting the nodes at TS side is valid. Because in the future may be we will introduce some kind of settings controlling the sort approach in the tree. And the TS (client) side should change the sort behavior based on the setting (which is also a client side stuff.)